content/embeds/supported-platforms-embed.md

```diff
@@ -5,55 +5,68 @@ Redis Enterprise Software is supported on several operating systems, cloud envi
 
-5. <a name="table-note-5" style="display: block; height: 80px; margin-top: -80px;"></a>See the [Redis Enterprise for Kubernetes documentation]({{<relref "/kubernetes">}}).
+The following table shows which Redis Enterprise Software version first tested and added support for each RHEL version:
 
-6. <a name="table-note-6" style="display: block; height: 80px; margin-top: -80px;"></a>Ubuntu 20.04 support was added in Redis Enterprise Software [6.4.2-43]({{<relref "/rs/release-notes/rs-6-4-2-releases/rs-6-4-2-43">}}).
+| RHEL version | Redis Enterprise version |
+|--------------|--------------------------|
+| 8.4 | 6.2.8 |
+| 8.5 | 6.2.10 |
+| 8.6 | 6.2.10 |
+| 8.7 | 6.4.2 |
+| 8.8 | 6.4.2 |
+| 8.9 | 7.2.4 |
+| 9.3 | 7.4.2 |
 
-7. <a name="table-note-7" style="display: block; height: 80px; margin-top: -80px;"></a>A release candidate for Amazon Linux 2 support was added in Redis Enterprise Software [6.4.2-61]({{<relref "/rs/release-notes/rs-6-4-2-releases/rs-6-4-2-61">}}). Official support for Amazon Linux 2 was added in Redis Enterprise Software [6.4.2-69]({{<relref "/rs/release-notes/rs-6-4-2-releases/rs-6-4-2-69">}}).
+## Operating system compatibility policy
 
-8. <a name="table-note-8" style="display: block; height: 80px; margin-top: -80px;"></a>Redis Enterprise Software [6.4.2-103]({{<relref "/rs/release-notes/rs-6-4-2-releases/rs-6-4-2-103">}}) and later supports RHEL 8.8.
+Redis maintains a list of [supported operating systems](#supported-platforms) for each major version of Redis Enterprise Software and the specific OS versions tested with Redis Enterprise releases. Since we update the list as new OS versions are introduced and old ones become obsolete, we encourage users to check the list and plan upgrades accordingly. We also suggest users keep Redis Enterprise and corresponding supported OS versions up to date.
 
-9. <a name="table-note-9" style="display: block; height: 80px; margin-top: -80px;"></a>Redis Enterprise Software [7.2.4-105]({{<relref "/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-105">}}) and later supports RHEL 8.9.
+We thoroughly test each supported major OS's most recent minor version to ensure the best compatibility and performance with every Redis Enterprise release. This process helps detect and address potential compatibility issues early on.
+
+Due to the vast array of minor updates and variations across operating systems, we cannot test compatibility with every minor OS version and each Redis Enterprise version. However, since OS vendors each have an Application Binary Interface (ABI) they support and avoid breaking, except to address severe security issues, newer minor OS versions are generally expected to work correctly. We will add a note to this document if specific OS minor versions have significant ABI or dependency changes. An earlier OS minor version also might work, although it is not guaranteed.
```

Comment:
```suggestion
Due to the vast array of minor updates and variations across operating systems, we cannot test compatibility with every minor OS version and each Redis Enterprise version. However, because OS vendors each have an Application Binary Interface (ABI) they support and avoid breaking, except to address severe security issues, newer minor OS versions are generally expected to work correctly. We will add a note to this document if specific OS minor versions have significant ABI or dependency changes. An earlier OS minor version also might work, although it is not guaranteed.
```